shared/app.js

```diff
@@ -92,10 +93,11 @@ module.exports = Backbone.Model.extend({
       this.templateAdapter = this.options.templateAdapterInstance;
     } else {
       var templateAdapterModule = attributes.templateAdapter || this.defaults.templateAdapter,
-      templateAdapterOptions = {entryPath: entryPath};
+        templateAdapterOptions = {entryPath: entryPath},
+        templateEngine = require(this.defaults.templateEngine);
```

Comment:
i don't think we'd want to use the defaults here - we should be able to override this.